crates/net/network/src/error.rs

```diff
@@ -214,7 +215,19 @@ impl SessionError for PendingSessionHandshakeError {
     fn is_fatal_protocol_error(&self) -> bool {
         match self {
             Self::Eth(eth) => eth.is_fatal_protocol_error(),
-            Self::Ecies(_) => true,
+            Self::Ecies(err) => matches!(
```

Comment:
we also need the same match in `merits_discovery_ban`:

https://github.com/paradigmxyz/reth/blob/main/crates/net/network/src/error.rs#L210-L210

Reply:
Done